docs/checks/reference/git.md

```diff
@@ -0,0 +1,18 @@
+Git
+===
+
+| Name            | Type    | Required   | Description                                           |
+| --------------- | ------- | ---------- | ----------------------------------------------------- |
+| Host            | String  | Y          | IP or FQDN the remote repository is located           |
+| Repository      | String  | Y          | The path to the remote repository                     |
+| Branch          | String  | Y          | The branch to clone from the repository               |
+| Port            | Integer | N :: 80    | The port to connect to for cloning the repository     |
+| HTTPS           | Boolean | N :: false | Whether to use HTTP or HTTPS                          |
+| HTTPSValidate   | Boolean | N :: false | Whether HTTPS certificates should be validated        |
+| Username        | String  | N          | Username to use for private repositories              |
+| Password        | String  | N          | Password for the user                                 |
+| ContentMatch    | Boolean | N :: false | Whether to check the contents of a file               |
+| ContentFile     | String  | N          | The path of the file to check the contents of         |
```

Comment:
Please add a short description at the bottom of the this file that explains how the path is expected. I am assuming it will be interpreted from the _root_ of the repository we are cloning but documenting whether something like `/path/to/file` works as well as `./path/to/file` or just one path format is needed here to prevent  user frustration.

Reply:
Added: https://github.com/scorestack/scorestack/pull/346/commits/f90b9d9858c1931f09160ee256a53cb5dc50a5cf